src/http_connection.cpp

```diff
@@ -373,8 +373,11 @@ void http_connection::start(std::string const& hostname, int port
 #if defined TORRENT_ASIO_DEBUGGING
 			add_outstanding_async("http_connection::on_i2p_resolve");
```

Comment:
this add_outstanding_async() belongs together with the call to async_name_lookup